### PR TITLE
Created Unfollow Route 

### PR DESCRIPTION
**Pull request by dhruvdabhi101**
Closes Issue no. #14 
Please feel free to give suggestions.

### CODE REVIEW

File: backend/routes/unfollow.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+  // unfollow with UserId
+  try {
+    const { data } = await refreshedClient.v2.unfollow(
+      twitterAccountId,
+      targetUserId
+    );
+      res.send(data);
+  } catch (error) {
+    console.log(error);
```
Comment (sanyamjain04) [inline backend/routes/unfollow.js:20]: Can you remove the console.log in line 20.

Comment (dhruvdabhi101) [reply backend/routes/unfollow.js:20]: yes sure.

**Comment by dhruvdabhi101** [comment]
Is it okay now ?

**Review by sanyamjain04 — APPROVED** [review]
✅ LGTM
Thanks, @dhruvdabhi101 for contributing to Kubetweet. 👏🏻
__
Summary: 👌 This code looks good.

**Pull request merged**